test/test_plugin.py

```
@@ -830,14 +833,80 @@ def module() -> Any:  # ruff:ignore[any-type]
+        return circuit_body()
+
+    @apply_pass("qco-to-catalystquantum")
+    @apply_pass("qc-to-qco")
+    @apply_pass("qco-to-qc")
+    @apply_pass("catalystquantum-to-qco")
+    @qml.qnode(get_device("lightning.qubit", wires=2))
+    def circuit() -> qml.measurements.ExpectationMP:
+        return circuit_body()
+
+    @qml.qjit(pass_plugins={MQT_PLUGIN_PATH}, dialect_plugins={MQT_PLUGIN_PATH})
+    def executable() -> Any:  # ruff:ignore[any-type]
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `executable`

(ANN401)